.travis.yml

```diff
@@ -1,138 +1,61 @@
 language: python
 dist: xenial
-
-matrix:
-  include:
-    - python: "3.6"
-      env: TOXENV=py36-django111-pil
-    - python: "3.6"
-      env: TOXENV=py36-django111-imagemagick
-    - python: "3.6"
-      env: TOXENV=py36-django111-graphicsmagick
-    - python: "3.6"
-      env: TOXENV=py36-django111-redis
-    - python: "3.6"
-      env: TOXENV=py36-django111-wand
-    - python: "3.6"
-      env: TOXENV=py36-django111-dbm
-    - python: 3.6
-      env: TOXENV=flake8
-
-    - python: "3.6"
-      env: TOXENV=py36-django20-pil
-    - python: "3.6"
-      env: TOXENV=py36-django20-imagemagick
-    - python: "3.6"
-      env: TOXENV=py36-django20-graphicsmagick
-    - python: "3.6"
-      env: TOXENV=py36-django20-redis
-    - python: "3.6"
-      env: TOXENV=py36-django20-wand
-    - python: "3.6"
-      env: TOXENV=py36-django20-dbm
-
-    - python: "3.6"
-      env: TOXENV=py36-django21-pil
-    - python: "3.6"
-      env: TOXENV=py36-django21-imagemagick
-    - python: "3.6"
-      env: TOXENV=py36-django21-graphicsmagick
-    - python: "3.6"
-      env: TOXENV=py36-django21-redis
-    - python: "3.6"
-      env: TOXENV=py36-django21-wand
-    - python: "3.6"
-      env: TOXENV=py36-django21-dbm
-
-    - python: "3.6"
-      env: TOXENV=py36-django30-pil
-
-    - python: "3.7"
-      env: TOXENV=py37-django20-pil
-    - python: "3.7"
-      env: TOXENV=py37-django21-pil
-
-    - python: "3.7"
-      env: TOXENV=py37-django22-pil
-    - python: "3.7"
-      env: TOXENV=py37-django22-imagemagick
-    - python: "3.7"
-      env: TOXENV=py37-django22-graphicsmagick
-    - python: "3.7"
-      env: TOXENV=py37-django22-redis
-    - python: "3.7"
-      env: TOXENV=py37-django22-wand
-    - python: "3.7"
-      env: TOXENV=py37-django22-dbm
-
-    - python: "3.7"
-      env: TOXENV=py37-django30-pil
-    - python: "3.7"
-      env: TOXENV=py37-django30-imagemagick
-    - python: "3.7"
-      env: TOXENV=py37-django30-graphicsmagick
-    - python: "3.7"
-      env: TOXENV=py37-django30-redis
-    - python: "3.7"
-      env: TOXENV=py37-django30-wand
-    - python: "3.7"
-      env: TOXENV=py37-django30-dbm
-
-    - python: "3.8"
-      env: TOXENV=py38-django30-pil
-    - python: "3.8"
-      env: TOXENV=py38-django30-imagemagick
-    - python: "3.8"
-      env: TOXENV=py38-django30-graphicsmagick
-    - python: "3.8"
-      env: TOXENV=py38-django30-redis
-    - python: "3.8"
-      env: TOXENV=py38-django30-wand
-    - python: "3.8"
-      env: TOXENV=py38-django30-dbm
-
+python:
+  - 3.6
+  - 3.7
+  - 3.8
+env:
```

Comment:
specifying the matrix this way runs tests on all supported environment combinations